controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -453,11 +460,16 @@ public String getScopeName() {
                                         Throwable ex = ExceptionHelpers.getRealException(e);
                                         if (ex instanceof StoreException.DataNotEmptyException) {
                                             // cant delete as there are transactions still running under epoch node
+                                            log.debug("stream {}/{} epoch {} not empty", scope, name, epoch);
                                             result.complete(false);
                                         } else {
+                                            log.warn("stream {}/{} deleting epoch {} threw exception {}", scope, name, epoch, ex.getClass().getName());
```

Comment:
yes, but that would put the entire stack on the logs. since this is at warn level and we know exactly where the error came by simply looking at the log, we dont need the entire stack posted.